framework.py

```diff
@@ -22,5 +20,19 @@ def business_entry():
 def shopping_bag():
     return render_template("Shopping_bag.html")
 
+@app.route("/POC")
+def poc():
+    Store_name = "Nike"
+    Categories = ["Men", "Woman", "Boys", "Girls", "Baby"]
+    Items=["Shirt", "Pants"]
+    Sizes=["S", "M", "L"]
+    return render_template("POC page.html", store_name=Store_name, categories=Categories, items=Items, sizes=Sizes)
+
+
+@app.route("/add_to_shopping_bag")

```

Comment:
you should read about routes in REST API. This is an article that explains about it: https://medium.com/@rajat_m/what-are-restful-routes-and-how-to-use-them-929129ae7bf6 
And I'll explain it anyway in the meeting today :)